model/NewSqlResultStorage.php

```diff
@@ -48,11 +49,11 @@ protected function prepareVariableDataForSchema($deliveryResultIdentifier, $test
         $persistence = $this->getPersistence();
 
         $createdAt = $this->getDateFromMicroTime($variable->getCreationTime());
-        
+
         return [
             self::VARIABLES_TABLE_ID => $persistence->getUniquePrimaryKey(),
             self::VARIABLES_FK_COLUMN => $deliveryResultIdentifier,
-            self::TEST_COLUMN => $test,
+            self::TEST_COLUMN => 'deprecated',
```

Comment:
How is this change going to work for us?

Reply:
The long story short - this is technical dept workaround related to the way how we store test and item variables, `null` and data utilisation from this column; @jbout could probably tell you some fascinating stories about this staff. 

But `deprecated` sets here to show that we are not using this column at all, but to remove it from the schema keeping in mind backward compatibility is too complex;